### ISSUE TITLE
Display hints to solve issues

### ISSUE THREAD

**Issue by Atinux** [nuxt-community/html-validator-module#1]
### Is your feature request related to a problem? Please describe.

A problem is always nicer with a solution attached to it.

### Describe the solution you'd like to see

It would be nice to map each issue to a hint / solution that we can display.

**Comment by nathanchase** [comment]
Maybe it would be easy enough to just add a full URL link to the formatter's output so that the error's corresponding rule links to the rule page at https://html-validate.org/ where the correct code example is shown?

Something like:
```
error: No space between attributes (attr-spacing) at inline:1:21:
> 1 | <input type="submit"class="foo">
    |                     ^^^^^
Details: https://html-validate.org/rules/attr-spacing.html
```

**Comment by Atinux** [comment]
This would be a nice first step for sure 👍 

PR welcome @nathanchase !

**Comment by nathanchase** [comment]
Here's a PR: https://github.com/nuxt-community/html-validator-module/pull/12

Output looks like this:
![image](https://user-images.githubusercontent.com/5963574/110383563-75e73300-802a-11eb-89e9-4c7eb9a3bfa0.png)

**Comment by danielroe** [comment]
closed in #12
